README.md

```diff
@@ -25,51 +25,68 @@ Next up, the service provider must be registered:
 ```php
 'providers' => [
     ...
-    Spatie\FailedJobMonitor\FailedJobMonitorServiceProvider::class,
+    Spatie\FailedJobMonitor\ServiceProvider::class,
```

Comment:
I prefer the old name. Could you rename the service provider back to `FailedJobMonitorServiceProvider`?